config/config.js

```diff
@@ -0,0 +1,15 @@
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+if (process.env.NODE_ENV === 'production') {
```

Comment:
I think it will be better to consolidate the environments together and load the development configuration if .env is not found. This is to ensure that we are not forced to use the default you defined (irrespective of the environment) and also caters for other environments (e.g STAGING and TEST). Something like this will be more productive imho.

```suggestion
module.exports = {
    DB_URL: process.env.DB_URL || 'mongodb://localhost:27017/ballers',
    PORT: process.env.PORT || 3000,
  };
```

Reply:
the `DB_URL` localhost would have to come first, or we'd be pushing to the production db everytime